unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/SettingsPanelHUDView.cs

```diff
@@ -198,7 +198,7 @@ public void OnAddHelpAndSupportWindow()
         public void SetWorldPreviewActive(bool isActive)
         {
             // TODO (Santi): Delete this line! The World Preview window will be deactivated until we decide (from design) how to show it.
-            isActive = false;
+            //isActive = false;
```

Comment:
Please remove commented code, commented code should not go into master

Reply:
good catch, thanks!

Reply:
Had to explicitly set it to false otherwise it would have been active